networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -463,9 +536,9 @@ def _set_vrf(self, ri, port, config):
 
     def _set_vrf_pid(self, ri, port, config):
         # Convert a VRF to a process iD for a router instance
-        vrf = self._get_vrf_name(ri)
+        vrf = self._get_vrf_name(ri).strip('nrouter-')[:6]
```

Comment:
_get_vrf_name provides either:

nrouter-<6 characters>

Or

nrouter-<6 characters>-<region ID>

We strip nrouter because it provides no new entropy for the hash, and use just the 6 VRF characters because we don't know expect the region ID to be present.